torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -612,6 +620,8 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 if not (hasattr(node, "meta") and "val" in node.meta and hasattr(node.meta["val"], "size")):
                     return None
                 input_tensor_shape = args[0].meta["val"].size()
+                if input_tensor_shape == torch.Size([]):
+                    input_tensor_shape = torch.Size([1])
```

Comment:
can you clarify the need for this?

Reply:
below code cannot handle `[]`, and the result of expand `[]` and `[1]` is the same, so I see `[]` as `[1]`